hr_timesheet_third_party/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, _
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    third_party_timesheet_ids = fields.One2many('account.analytic.line', 'third_party_id')
+    third_party_unit_amount_total = fields.Integer(string="Third Party Hours", compute="_compute_third_party_unit_amount_total")
```

Comment:
That's wrong IMO becoause `unit_amount` of account.analytic.line is `float`, then this field should be `float` too.

Reply:
Yes it should be float but at the time of displaying it only wanted to see the hours without rounding so I opted for this option.

Reply:
OK